pr_inspector/services/github_service.py

```
@@ -24,6 +29,27 @@ class PrDetails:
+        output.append(f"Title: {self.pr_title}\n")
+        output.append(f"Body: {self.pr_body.strip() if self.pr_body else ''}\n")
+        output.append("\n=== Files Changed ===")
+        for pr_file in self.pr_files:
+            output.append(f"- {pr_file.file_name}:")
+            # TODO: see if we should truncate or not. Currently truncating
+            # for testing urposes, might change later.
+            if pr_file.file_diff is not None:
+                diff_snippet = pr_file.file_diff[:MAX_DIFF_LENGTH]
+                # add ellipsis if truncated
+                if len(pr_file.file_diff) > MAX_DIFF_LENGTH:
+                    logger.info(f"Diff for {pr_file.file_name} was truncated to {MAX_DIFF_LENGTH} characters.")
```

Ruff (0.14.10)

Logging statement uses f-string

(G004)